### DISCUSSION TITLE
cephfs - inodes available as "-1"

### DISCUSSION THREAD

**Discussion by reefland** [rook/rook#11637] [Q&A] [answered]
CephFS PV as seen by Prometheus is showing "-1" available inodes, which triggers an alert `PersistentVolumeInodes are filling up.  The PersistentVolume claimed by <claimName> in Namespace <namespace> only has 0% free inodes.`

In searching this, it seems that cephfs dynamically allocates inodes but I don't see a reason for reporting `-1` vs nothing (no alert). 

So before I setup an alert silence for this, just wanted to make sure that is expected and I'm not silencing something I shouldn't.

Thanks.

**Comment by Madhu-1** [comment, marked as answer]
@reefland yes you are correct its dynamic allocation and its a false alert and fixed in https://github.com/ceph/ceph-csi/pull/3407, in next cephcsi release you will not gets this reported

**Reply by reefland** [reply]
Thank you, I will setup an Alert Silence for now.
